src/FormatNegotiator.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Franzl\Middleware\Whoops;
+
+use Psr\Http\Message\ServerRequestInterface;
+
+/**
+ * Middleware returns the client preferred format.
+ */
+class FormatNegotiator
+{
+    /**
+     * @var array Available formats with the mime types
+     */
+    private static $formats = [
+        'html' => ['text/html', 'application/xhtml+xml'],
+        'css' => ['text/css'],
+        'gif' => ['image/gif'],
+        'png' => ['image/png', 'image/x-png'],
+        'jpg' => ['image/jpeg', 'image/jpg'],
+        'jpeg' => ['image/jpeg', 'image/jpg'],
+        'json' => ['application/json', 'text/json', 'application/x-json'],
+        'jsonp' => ['text/javascript', 'application/javascript', 'application/x-javascript'],
+        'js' => ['text/javascript', 'application/javascript', 'application/x-javascript'],
+        'pdf' => ['application/pdf', 'application/x-download'],
+        'rdf' => ['application/rdf+xml'],
+        'rss' => ['application/rss+xml'],
+        'atom' => ['application/atom+xml'],
+        'xml' => ['text/xml', 'application/xml', 'application/x-xml'],
+        'txt' => ['text/plain'],
+        'mp4' => ['video/mp4'],
+        'otf' => ['font/opentype'],
+        'ttf' => ['font/ttf', 'application/font-ttf', 'application/x-font-ttf'],
+        'ogg' => ['audio/ogg'],
+        'ogv' => ['video/ogg'],
+        'webm' => ['video/webm'],
+        'woff' => ['font/woff', 'application/font-woff', 'application/x-font-woff'],
+        'webp' => ['image/webp'],
+        'svg' => ['image/svg+xml'],
+        'zip' => ['application/zip', 'application/x-zip', 'application/x-zip-compressed'],
+    ];
```

Comment:
Nah, just remove them. I wonder... should we fallback to txt or html by default? I'm leaning towards the former...